apps/meteor/client/lib/voip/VoIPUser.ts

```diff
@@ -633,6 +653,29 @@ export class VoIPUser extends Emitter<VoipEvents> {
 		throw new Error('Something went wrong');
 	}
 
+	/* Helper routines for checking call actions BEGIN */
+
+	private canRejectCall(): boolean {
+		if (this._callState !== 'OFFER_RECEIVED' && this._callState !== 'OFFER_SENT') {
+			return false;
+		}
+		return true;
```

Comment:
```suggestion
		return ['OFFER_RECEIVED', 'OFFER_SENT'].includes(this._callState);
```
Or maybe you can avoid if-else altogether here!

Reply:
Fixed.

Reply:
Fixed.